app/views/organisations/edit.html.erb

```diff
@@ -29,6 +29,33 @@
         autocomplete: "tel",
         width: 20 %>
 
+      <% if current_user.support? %>
+        <%= f.govuk_radio_buttons_fieldset :group_member,
+           legend: { text: "Is this organisation part of a housing provider group structure?", size: "m" } do %>
+          <%= f.govuk_radio_button :group_member, true,
+             label: { text: "Yes" },
+             "data-controller": "conditional-question",
+             "data-action": "click->conditional-question#displayConditional",
+             "data-info": { conditional_questions: { group: [true] }, type: "organisation" }.to_json do %>
+            <%= f.govuk_collection_select :group_member_id,
+                group_organisation_options,
+                :id,
+                :name,
+                label: { text: "Search for an organisation that is part of the same group as this organisation", size: "m" },
+                options: { disabled: [""], selected: @organisation.oldest_group_member&.id || "" },
```

Comment:
Concerned that this name doesn't match to what's being selected here - the user is just picking _any_ org in the same group, not necessarily the oldest one.

Reply:
If we're getting rid of `group_member_id` we'll want to change this back to `oldest_group_member` 😅